invenio.cfg

```diff
@@ -211,15 +223,23 @@ OAUTHCLIENT_CERN_USER_INFO_FROM_ENDPOINT = False
 
 handlers = _keycloak_helper.get_handlers()
 handlers["signup_handler"]["info_serializer"] = cern_info_serializer
-rest_handlers = _keycloak_helper.get_handlers()
-rest_handlers["signup_handler"]["info_serializer"] = cern_info_serializer
+handlers["signup_handler"]["groups_serializer"] = cern_group_serializer
+handlers["signup_handler"]["groups"] = cern_group_handler
+rest_handlers = _keycloak_helper.get_rest_handlers()
+handlers["signup_handler"]["info_serializer"] = cern_info_serializer
+rest_handlers["signup_handler"]["groups_serializer"] = cern_group_serializer
+rest_handlers["signup_handler"]["groups"] = cern_group_handler
 
 OAUTHCLIENT_SIGNUP_FORM = confirm_registration_form
 
+_remote_app_name = "cern"
+
 OAUTHCLIENT_REMOTE_APPS = {
-    "cern": _keycloak_helper.remote_app,
+    _remote_app_name: _keycloak_helper.remote_app,
 }
 
+SECURITY_POST_LOGOUT_VIEW = "/oauth/{}/logout".format(_remote_app_name)
```

Comment:
This works only for one remote, any app with more remotes should be able register it's own logout url, to be checked if it makes sense to override ``SECURITY_LOGOUT_URL``